.eslintrc

```diff
@@ -48,6 +48,8 @@
           "ClassDeclaration": true
         }
       }
-    ]
+    ],
+    "object-curly-newline": ["error", { "multiline": true, "minProperties": 4 }],
```

Comment:
@filschristian  We need this Eslint rule for better readability of long statements. Try to fix the issue with Daniel

Reply:
this rule does not have anything related to long statements!

Reply:
@filschristian What is the use of the rule that you added?

Reply:
@oesukam this line is for object's curly brackets

Reply:
@filschristian Ok. But I have seen that you have removed some rules

Reply:
@oesukam So I removed the max-line rule as you suggested but I wanted to let you know that this line you commented on was not the right one. You should have commented on max-line.

Reply:
@filschristian Ok. Thanks for the feedback